docs/ref/models/relations.txt

```diff
@@ -46,6 +46,10 @@ Related objects reference
             >>> e = Entry.objects.get(id=234)
             >>> b.entry_set.add(e) # Associates Entry e with Blog b.
 
+        If you have multiple objects to add it is more efficient to use
+        :ref:`a single call to add <insert-many-to-many-in-bulk>`
+        rather than multiple calls.
+
```

Comment:
I'm not sure about adding this (or the same for delete below). We already have `*odjs` in the signature, `specified model objects` (plural) in the text above, and uses in the [introductory M2M example](https://docs.djangoproject.com/en/2.2/topics/db/examples/many_to_many/).

It feels a bit out-of-place here. _"more efficient"_ being the exact topic of `optimization.txt`.

Reply:
I thought having a reference to the optimization section related to these methods would improve their discoverability of the optimization section.

Reply:
I agree with Carlton, and I'm going to revert this change.